src/OpenTelemetry/Logs/ILogger/OpenTelemetryLoggingExtensions.cs

```diff
@@ -50,15 +50,40 @@ public static class OpenTelemetryLoggingExtensions
     /// <returns>The supplied <see cref="ILoggingBuilder"/> for call chaining.</returns>
     public static ILoggingBuilder AddOpenTelemetry(
         this ILoggingBuilder builder)
+        => AddOpenTelemetryInternal(builder, configureBuilder: null, configureOptions: null);
+
+    /// <summary>
+    /// Adds an OpenTelemetry logger named 'OpenTelemetry' to the <see cref="ILoggerFactory"/>.
+    /// </summary>
+    /// <remarks><inheritdoc cref="AddOpenTelemetry(ILoggingBuilder)" path="/remarks"/></remarks>
+    /// <param name="builder">The <see cref="ILoggingBuilder"/> to use.</param>
+    /// <param name="configure">Optional configuration action.</param>
+    /// <returns>The supplied <see cref="ILoggingBuilder"/> for call chaining.</returns>
+    public static ILoggingBuilder AddOpenTelemetry(
+        this ILoggingBuilder builder,
+        Action<OpenTelemetryLoggerOptions>? configure)
+        => AddOpenTelemetryInternal(builder, configureBuilder: null, configureOptions: configure);
+
+    private static ILoggingBuilder AddOpenTelemetryInternal(
+        ILoggingBuilder builder,
+        Action<LoggerProviderBuilder>? configureBuilder,
```

Comment:
Was it intentional to add the parameter `Action<LoggerProviderBuilder>` in this PR? Or was this supposed to go with the cleanup of Logs stuff from this PR?

Reply:
I refactored this for `UseOpenTelemetry` which was later removed but I kept the refactoring. I would prefer to keep this because I'll use it on my next PR adding back some of that logging stuff (it will just save me a bit of time). LMK if you really want me to remove it 🥲

Reply:
We can keep this.